File: src/BuildMaestro.Web/wwwroot/app/models/git-commit.model.ts
export class GitCommitModel implements IGitCommitModel {
    public author: string;
    public dateTime: string;
    public hash: string;
    public message: string;
}

export interface IGitCommitModel {
    author: string;
    dateTime: string;
    hash: string;
    message: string;
}

File: src/BuildMaestro.Shared/Models/GitCommitModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace BuildMaestro.Shared.Models
{
    public class GitCommitModel
    {
        public string Author { get; set; }

        public string DateTime { get; set; }

        public string Hash { get; set; }

        public string Message { get; set; }
    }
}
